doc/source/singlevisit.rst

```diff
@@ -505,6 +518,54 @@ then gets used to define the shape, size and WCS pointing for all drizzle produc
 taken with the same detector in the visit.
 
 
+Handling Special Images
+-------------------------
+Grism and Prism images are acquired as part of a visit, in conjunction with their
+direct image counterparts, and classified as spectroscopic data.  It is beneficial for these
+images to share a common WCS with the corresponding direct images from the same detector 
+in the visit.  Because the Grism/Prism data cannot be used in the alignment procedure due
+to the nature of the data, the best WCS solution that can be generated for these images
+is an **a priori** solution. An **a priori** solution has been determined for essentially
+all HST data by correcting the coordinates of the guide stars that were used for the observation 
+to the coordinates of the same guide stars as determined by GAIA, in this case.  The actual 
+image pixels have not be used in the WCS determination.  The WCSNAME for this 
+**a priori** solution is of the form::
+
+  <Starting WCS>-<Astrometric Catalog>
+
+  For example,
+  'IDC_0461802ej-GSC240'
+
+
+where the **Astrometric Catalog** refers to the specific astrometric catalog used to correct 
+the guide star positions.  
```

Comment:
I am not sure where it would be most appropriate to add any further references, but it may be helpful to provide a pointer to the description of all the possible GSC catalog options.  MAST has a public page https://outerspace.stsci.edu/display/GC/Basic+Catalog+information that would make for a suitable reference.  If this should, instead, be added elsewhere in our documentation, we can put it in later.

Reply:
I think this is a great idea so I have added the reference in this location.